front/src/common/Pathfinding/Pathfinding.tsx

```diff
@@ -36,7 +36,7 @@ import {
 } from 'reducers/osrdconf/selectors';
 import infraLogo from 'assets/pictures/components/tracks.svg';
 import InfraLoadingState from 'applications/operationalStudies/components/Scenario/InfraLoadingState';
-import { Spinner } from '../Loader';
+import { Spinner } from '../Loaders';
```

Comment:
Should we import it with its full path in case Pathfinding moves after the refacto ?

Reply:
Sure ! Thanks ! :)